xdsl/dialects/llvm.py

```diff
@@ -95,6 +97,66 @@ def is_typed(self):
         return not isinstance(self.type, NoneAttr)
 
 
+@irdl_op_definition
+class GEPOp(Operation):
+    name = "llvm.getelementptr"
+
+    ptr: Annotated[Operand, LLVMPointerType]
+
+    ssa_indices: Annotated[VarOperand, IntegerType]
+
+    elem_type: OptOpAttr[Attribute]
+
+    rawConstantIndices: OpAttr[DenseArrayBase]
+
+    inbounds: OptOpAttr[UnitAttr]
+
+    result: Annotated[OpResult, LLVMPointerType]
+
+    @staticmethod
+    def get(ptr: SSAValue | Operation,
+            result_type: Attribute
+            | LLVMPointerType = LLVMPointerType.opaque(),
```

Comment:
Be careful with default arugments in Python. If you use mutable objects and mutate them, the next function call will get the mutated object.

It is fine in this case, because LLVMPointerType is an Attribute and, therefore, immutable. Just for future referene :)